make/migrations/postgresql/0050_2.2.0_schema.up.sql

```diff
@@ -288,3 +288,164 @@ ALTER TABLE scan_report DROP COLUMN IF EXISTS end_time;
 
 /*add unique for vendor_type+vendor_id to avoid dup records when updating policies*/
 ALTER TABLE schedule ADD CONSTRAINT unique_schedule UNIQUE (vendor_type, vendor_id);
+
+/*move the gc schedule job records into the new schedule table*/
+DO $$
+DECLARE
+    schd RECORD;
+    new_schd_id integer;
+    exec_id integer;
+    exec_status varchar(32);
+    task_status varchar(32);
+    task_status_code integer;
+BEGIN
+    FOR schd IN SELECT * FROM admin_job where job_name='IMAGE_GC' and job_kind='Periodic'
+    LOOP
+        INSERT INTO schedule (vendor_type, vendor_id, cron, callback_func_name,
+            callback_func_param, cron_type, extra_attrs, creation_time, update_time)
+            VALUES ('GARBAGE_COLLECTION', -1,
+                (SELECT cron_str::json->>'cron' from admin_job where id=schd.id),
```

Comment:
Why we need to query the record from the `admin_job` again rather than using the `schd`?

Reply:
I just want to segregate the logic. Do you mean to do these steps into one table loop?

Reply:
updated.